src/bin/common.rs

```diff
@@ -10,8 +10,8 @@
 use crate::muxer::{create_muxer, Muxer};
 use crate::{ColorPrimaries, MatrixCoefficients, TransferCharacteristics};
 use clap::{App, AppSettings, Arg, ArgMatches, SubCommand, Shell};
-use rav1e::partition::BlockSize;
 use rav1e::*;
+use scan_fmt::scan_fmt;
```

Comment:
this one is used only in the cli, is this the right place?

Reply:
yes, we could rename the filename so it is less misleading later :)